tests/monster_test.cpp

```diff
@@ -300,37 +300,57 @@ static void monster_check()
 TEST_CASE( "write_slope_to_speed_map_trig", "[.]" )
 {
     clear_map_and_put_player_underground();
+    bool old_trigdist = trigdist;
     override_option opt( "CIRCLEDIST", "true" );
     trigdist = true;
     test_moves_to_squares( "mon_zombie_dog", true );
     test_moves_to_squares( "mon_pig", true );
+    if( ! old_trigdist ) {
+        override_option opt( "CIRCLEDIST", "false" );
```

Comment:
I'm pretty sure this won't work, it'll immediately go out of scope and have no effect.

Reply:
So just set trigdist back to old_trigdist?

Reply:
> So once that variable goes out of scope, the destructor puts that option (as opposed to trigdist!) back to the old version?

Correct.